userspace/libraries/libc/string.cpp

```diff
@@ -62,10 +62,37 @@ void *memmove(void *dest, const void *src, size_t n)
     return dest;
 }
 
+#ifdef __x86_64__
```

Comment:
GCC is really smart and generates nearly the same code for the two functions by using 128bit MMX registers internally ;)